src/scripts/autoThemes.js

```diff
@@ -0,0 +1,14 @@
+import * as userSettings from 'userSettings';
```

Comment:
Not a fan of the name of this file. It could also probably be part of skinManager.

Reply:
I pulled this naming from `autoBackdrops` which I also hate. It was mainly to get a unirform name for those two weird files for now but we should probably remove them yeah.